R/bundle.R

```diff
@@ -853,6 +869,7 @@ addPackratSnapshot <- function(bundleDir, implicit_dependencies = c()) {
     # rethrow error so we still halt deployment
     stop(e)
   })
+  # if (verbose) logger(" - done performing packrat snapshot")
```

Comment:
Here and elsewhere -- it is not clear why this code is commented out; please either uncomment it or remove it.

Reply:
Fixed in latest commit